src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -766,6 +763,19 @@ protected DynamoModel(IStartConfiguration config)
             }
         }
 
+        public bool AddPackagePath(string path)
```

Comment:
can you add a summary to this method that details the behavior?

Reply:
Summary has been added.